extension/services/applemusic/applemusic.js

```diff
@@ -0,0 +1,12 @@
+import * as music from "../../intents/music/music.js";
+import MusicService from "../../background/musicService.js";
+
+class AppleMusic extends MusicService {}
+
+Object.assign(AppleMusic, {
+  id: "applemusic",
+  title: "AppleMusic",
```

Comment:
There should be a space, i.e., `"Apple Music"`

Reply:
Thanks, noted.